src/java.base/share/man/java.md

```diff
@@ -0,0 +1,4052 @@
+---
+# Copyright (c) 1994, 2024, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
```

Comment:
Agree.

Reply:
Yeah, sorry for that, and thanks for catching it. 

I just copied the header from a build file, since it was conveniently already formatted with the leading `#`. I just assumed that this was the pure GPL and did not check close enough to verify. But it turns out that most of the build system files actually have the classpath exception, which makes no sense. 

I've filed https://bugs.openjdk.org/browse/JDK-8344191 to deal with that.